spring-cloud-huawei-governance/pom.xml

```diff
@@ -100,6 +100,10 @@
       <artifactId>spring-cloud-gateway-server</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.servicecomb</groupId>
```

Comment:
governance module do not depend on service-center-client

Reply:
Modified